mev-relay-rs/src/relay.rs

```diff
@@ -448,6 +451,28 @@ impl Relay {
         state.auctions.insert(auction_request, auction_context);
         Ok(())
     }
+
+    fn verify_blinded_block_signature(
+        &self,
+        signed_block: &mut SignedBlindedBeaconBlock,
+    ) -> Result<(), ethereum_consensus::Error> {
```

Comment:
easier and cleaner to just follow the error pattern for everything else here, which just uses the `Error` type

it has a variant for `ethereum_consensus::Error` so you can leverage `Into/From` etc